js/stores/appStore.js

```diff
@@ -503,6 +503,26 @@ const handleAppAction = (action) => {
     case AppConstants.APP_SET_DICTIONARY:
       appState = appState.setIn(['dictionary', 'locale'], action.locale)
       break
+    case AppConstants.APP_CLEAR_DATA:
+      if (action.clearDataDetail.get('browserHistory')) {
+        handleAppAction({actionType: AppConstants.APP_CLEAR_SITES_WITHOUT_TAGS})
```

Comment:
This is the same as how we did it before but I posted in this issue to handle that in next release after this one. 
https://github.com/brave/browser-laptop/issues/3065